### PR TITLE
added doc to PaintTool

### PR DESCRIPTION
**Pull request by SalvatoreScaramuzzino**
Hi to everyone, hi added a little example of painttool usage. Is very useful for newbie to use this tool in order to get masks quickly.

I hope this can be useful.

### CODE REVIEW

**Comment by stefanv** [comment]
Thanks!  Currently, Travis tests are failing.

File: skimage/viewer/canvastools/painttool.py [hunk 1]
```diff
@@ -37,6 +37,24 @@ class PaintTool(CanvasToolBase):
...
+    ----------
+    >>> from skimage.data import camera
+    >>> import matplotlib.pyplot as plt
+    >>> from skimage.viewer.canvastools import PaintTool
+    >>> import numpy as np
+
+    >>> img = camera()
+
+    >>> ax = plt.subplot(111)
+    >>> plt.imshow(img, cmap=plt.cm.gray)
```
Comment (OrkoHunter) [inline skimage/viewer/canvastools/painttool.py]: I'm not sure but this line needs to be skipped too from the doctest.
From the travis log
```
File "/home/travis/build/scikit-image/scikit-image/skimage/viewer/canvastools/painttool.py", line 51, in skimage.viewer.canvastools.painttool.PaintTool
Failed example:
    plt.imshow(img, cmap=plt.cm.gray)
Expected nothing
Got:
    <matplotlib.image.AxesImage object at 0x7f70085f0390>
```

**Comment by SalvatoreScaramuzzino** [comment]
this looks strange for me...

**Comment by OrkoHunter** [comment]
@SalvatoreScaramuzzino Strange? Ah the coveralls? Well, we are used to it. :beers: 
Looks good. Now try [squashing your last 7 commits into 1 useful one](https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) :)

**Comment by SalvatoreScaramuzzino** [comment]
squashing seems doesn't be very helpful

**Comment by OrkoHunter** [comment]
Sorry being vague but there is some issue with coveralls and we can't do much about it. I suggested squashing because it helps in maintaining a good git history of the repository.

**Comment by emmanuelle** [comment]
@SalvatoreScaramuzzino thank you for the useful addition. It would be better if you could squash your commits as suggested above; however, if it seems too complicated to you, we can make an exception. But since you've already made useful contributions to scikit-image in the past, maybe you can try this squashing process so that you'll know it for your future contributions? Thanks!

**Comment by blink1073** [comment]
LGTM, thanks @SalvatoreScaramuzzino!

**Pull request merged**